### PR TITLE
Update license and add comment to readme about IE11

### LINKED ISSUE #45: Inconsistent License
https://github.com/KoryNunn/scroll-into-view/blob/ebeca5849654d2d525567e26d22ba56cf2950d48/package.json#L21

The license file attached is MIT-License but the package.json says BSD.

### PR DESCRIPTION
**Pull request by mwood23**
closes #45

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ scrollIntoView(someElement, {
     validTarget: function(target, parentsScrolled){
 
         // Only scroll the first two elements that don't have the class "dontScroll"
+        // Object.matches is not supported in IE11, consider using Lodash equivalent if you need to support that browser
```
Comment (KoryNunn) [inline README.md]: Please change this recommendation to: "consider using `matches-selector` module or equivalent if you need to support that browser"

**Comment by KoryNunn** [comment]
Thanks for the PR, If you can change the recommended polyfill I'll merge.

File: README.md (hunk 1 above)
Comment (KoryNunn) [reply README.md]: also change `Object.matches` -> `Element.matches`

Comment (mwood23) [reply README.md]: done!

**Pull request merged**